frontend/src/components/atoms/CorrelationDropdown.tsx

```diff
@@ -0,0 +1,109 @@
+import { ChevronDownIcon } from '@chakra-ui/icons';
+import { Button, Checkbox, Menu, MenuButton, MenuItem, MenuList } from '@chakra-ui/react';
+import React from 'react';
+import { CorrelationFilter } from '../../types/generalTypes';
+
+interface Props {
+  onChangeCorrelation: React.Dispatch<React.SetStateAction<CorrelationFilter>>;
+  positive: boolean;
+}
+
+const CorrelationDropdown = ({ onChangeCorrelation, positive }: Props) => (
+  <Menu closeOnSelect={false} closeOnBlur autoSelect={false}>
+    <MenuButton as={Button} color="cyan.600" rightIcon={<ChevronDownIcon />}>
+      {positive ? 'Positiv ' : 'Negativ '}
+      virkninger
+    </MenuButton>
+    <MenuList bg="cyan.500">
+      <MenuItem
+        _hover={{
+          bg: '#A1E3F0',
+        }}
+      >
+        <Checkbox
+          width="100%"
+          height="100%"
+          type="checkbox"
+          defaultIsChecked
+          color="white"
+          size="lg"
+          onChange={() => {
+            if (positive) {
+              onChangeCorrelation((current) => ({
+                ...current!,
+                pLow: !current?.pLow,
+              }));
+            } else {
+              onChangeCorrelation((current) => ({
+                ...current!,
+                nLow: !current?.nLow,
+              }));
+            }
+          }}
```

Comment:
I would add a null check on `current` instead of using the bang operator. 

```suggestion
          onChange={() => {
            if (current === null) return
            if (positive) {
              onChangeCorrelation((current) => ({
                ...current!,
                pLow: !current?.pLow,
              }));
            } else {
              onChangeCorrelation((current) => ({
                ...current!,
                nLow: !current?.nLow,
              }));
            }
          }}
```

You might have to check for `undefined` as well, but it is considered a better practice.

Reply:
I have set default values as true. So i can never be null or undefined.

Reply:
forgot to remove them